core/store/src/cold_storage.rs

```diff
@@ -157,22 +158,31 @@ pub fn update_cold_head<D: Database>(
 
 /// Copies all contents of all cold columns from `hot_store` to `cold_db`.
 /// Does it column by column, and because columns can be huge, writes in batches of ~`batch_size`.
+/// Returns:
+/// - Ok(`true`) if everything was copied;
+/// - Ok(`false`) if the process was interrupted via `keep_going` flag;
```

Comment:
nit, real smol, feel free to ignore me: A small enum instead of bool would be slightly nicer on the eyes.

Reply:
no prob